### ISSUE TITLE
Issue updating ARM template json for azure

### ISSUE THREAD

**Issue by hopchouinard** [microsoft/vscode-azurearmtools#390]
Repro steps:

1.
2.

Action: queryUpdateSchema
Error type: Error
Error Message: TextEditor#edit not possible on closed editors

Version: 0.8.2
OS: win32
Product: Visual Studio Code
Product Version: 1.40.0
Language: en

<details>
<summary>Call Stack</summary>

```
d.edit extensionHostProcess.js:493:41
AzureRMTools.<anonymous> extension.bundle.js:127:181867
extension.bundle.js:127:174871extension.bundle.js:127:174871
n extension.bundle.js:127:174619
AzureRMTools.replaceSchema extension.bundle.js:127:181575
AzureRMTools.<anonymous> extension.bundle.js:127:181275
o extension.bundle.js:127:174675
```

</details>

**Comment by StephenWeatherford** [comment]
@hopchouinard Thanks, this is being tracked in #389.  I'm assuming you switched tabs before clicking "Use latest" or else closed the editor.  If it was something else, please let me know.  Or if it doesn't work when the editor is still open, thanks!

**Comment by hopchouinard** [comment]
Hi Stephen,

I tried multiple ways, including switching tabs, close the editor, recreating the whole json file by hand. No matter which way I try, I always get the same error message.

Regards,

**Comment by StephenWeatherford** [comment]
@hopchouinard I need to make sure I'm understanding...  I've got a fix for the fact that if you switch tabs or close the editor before clicking "Use latest" that it does not work.

Does it work for you if you do *not* switch tabs or close the editor first?  As far as I know, that should always succeed.

If that doesn't work, please let me know.  You can also change the $schema manually to the newest version of deploymentTemplate.json:
`https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#`

None of the other schemas (subscription, management group, tenant) have more recent schemas available.

**Comment by hopchouinard** [comment]
I’m sorry for the confusion, yes it does work if I do not switch tabs.
What I wanted to say was that closing the editor and restarting or recreating a new json file had no impact if I switch tab before clicking "Use latest".

**Comment by StephenWeatherford** [comment]
Okay, thanks!  I really appreciate your taking the time for the questions.  I have a fix for this (#392). In the meantime you can edit manually or just make sure to make the template active before clicking Use Latest.

**Comment by StephenWeatherford** [comment]
Fixed by #392
